{"name": "test_update_user_profile_failed", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <test_user_service.TestUserService object at 0x000002635187AAD0>\n\n    @pytest.mark.update_user_profile_failed\n    def test_update_user_profile_failed(self):\n        try:\n            update_user_profile('test_user_001_lisa', 'test_user_Justin', 'Justin@cc.cc', '2002-02-03')\n>           assert False\nE           assert False\n\ntest\\test_user_service.py:133: AssertionError"}, "attachments": [{"name": "log", "source": "4b733a8a-3b71-44c9-a2bc-4335e9f0935a-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "abf8d065-b8ff-4695-9e20-d7c0155fa1bf-attachment.txt", "type": "text/plain"}], "start": 1703060046381, "stop": 1703060046385, "uuid": "ab248a42-2b85-4543-887b-70fe8e18936a", "historyId": "d2aefa49bdbff7a42b8a6bb92744536f", "testCaseId": "d2aefa49bdbff7a42b8a6bb92744536f", "fullName": "test.test_user_service.TestUserService#test_update_user_profile_failed", "labels": [{"name": "tag", "value": "update_user_profile_failed"}, {"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_user_service"}, {"name": "subSuite", "value": "TestUserService"}, {"name": "host", "value": "LAPTOP-SJL5AU2K"}, {"name": "thread", "value": "59520-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_user_service"}]}